Makefile

```diff
@@ -20,7 +21,7 @@ fmt: **/*.go ## Format code
 	go fmt ./...
 
 tools: ## Install additional required tooling
-	go get -u golang.org/x/lint/golint
+	GOFLAGS= go install golang.org/x/lint/golint
```

Comment:
Can you explain this change? The [golint readme](https://github.com/golang/lint#installation) recommends `go get`.

Reply:
as far as i've seen with my use of go modules, `go get` changes the required version of a dependency (see https://blog.golang.org/using-go-modules).

since [the `golint` version is in the `go.mod` file](https://github.com/flosell/iam-policy-json-to-terraform/pull/8/files#diff-37aff102a57d3d7b797f152915a6dc16R9), i was trying to avoid changing it

if you want it to upgrade each time, we can use a different method of install too

Reply:
ok, makes sense to me!